packages/js/test/plugins/auctionHouseModule/withdrawFromTreasuryAccount.test.ts

```diff
@@ -0,0 +1,73 @@
+import test, { Test } from 'tape';
+import { sol } from '@/types';
+import {
+  metaplex,
+  killStuckProcess,
+  createNft,
+  createWallet,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+
+killStuckProcess();
+
+test('[auctionHouseModule] withdraw from treasury account on an Auction House', async (t: Test) => {
+  // Given we have an Auction House with fee that equals 100 basis points and an NFT.
+  const mx = await metaplex();
+  const buyer = await createWallet(mx);
+  const nft = await createNft(mx);
+
+  const { auctionHouse } = await createAuctionHouse(mx, null, {
+    sellerFeeBasisPoints: 1000,
+  });
+
+  // And we deposited 2 SOL to the buyer's escrow, so 0.1 SOL fee can be withdrawn.
```

Comment:
With 10% fees, we should be able to take 0.2 SOL. Could you update the rest of the test accordingly?

```suggestion
  // And we deposited 2 SOL to the buyer's escrow, so 0.2 SOL fee can be withdrawn.
```

Note that the fee is taken from the seller. So the buyer doesn't have to have more SOLs to pay for the fees. The buy's deposit account and the listing price can be exactly the same amount.